stdlib/public/core/Span/RawSpan.swift

```diff
@@ -0,0 +1,702 @@
+//===--- RawSpan.swift ----------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// A RawSpan represents a span of initialized memory
+// of unspecified type.
+@_disallowFeatureSuppression(NonescapableTypes)
+@available(SwiftStdlib 9999, *)
```

Comment:
Should we use `SwiftStdlib 5.1` here, or are we intentionally holding off on that pending the "how do we create an instance of a `Span` discussion?"

Reply:
I think `SwiftStdlib 6.1` would be fine, but we need to make sure to remember to change it if the feature slips to 6.2 with delays to the other proposals.

Reply:
Okay!